selfdrive/navd/map_renderer.cc

```diff
@@ -62,7 +64,7 @@ MapRenderer::MapRenderer(const QMapLibre::Settings &settings, bool online) : m_s
   std::string style = util::read_file(STYLE_PATH);
   m_map.reset(new QMapLibre::Map(nullptr, m_settings, fbo->size(), 1));
   m_map->setCoordinateZoom(QMapLibre::Coordinate(0, 0), DEFAULT_ZOOM);
-  m_map->setStyleJson(style.c_str());
```

Comment:
why doesn't this work?

Reply:
actually it's working fine, the 404 errors were coming from the number of frames needed to pass from valid to invalid and vice-versa (below)